### PR TITLE
Improved the Signup page using Material UI.

### PR DESCRIPTION
**Pull request by rak-shit**
Screenshot 2020-03-05 at 9 26 11 PM
Screenshot 2020-03-05 at 9 26 20 PM

### CODE REVIEW

File: src/auth/login/login.scss [hunk 1]
```diff
@@ -57,3 +57,32 @@
...
+        font-size: 12px;  
+        line-height: 15px;
+      }
+      a {
+        button {
+          background: #1A73E8;
+          box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.08);
+          border-radius: 5px;
+          .loginbtn-content {
+            font-family: Inter;
```
Comment (vaibhavdaren) [inline src/auth/login/login.scss]: This is not a good way to use fonts

Comment (devesh-verma) [reply src/auth/login/login.scss]: @rak-shit in scss you have `scss variable` please make use of that.
[1]. https://sass-lang.com/documentation/variables

Comment (rak-shit) [reply src/auth/login/login.scss]: Okay. I will see that.

Comment (rak-shit) [reply src/auth/login/login.scss]: > This is not a good way to use fonts

Hey @vaibhavdaren, can you please elaborate a bit. I mean are you saying about the usage of fonts for each element separately or the way I am using it for each element.

**Comment by rak-shit** [comment]
> check why netlify is failing for this PR.

sure

File: src/auth/login-form/login-form.js [hunk 2]
```diff
@@ -55,28 +57,34 @@ class LoginForm extends Component {
...
-              required
-              type="password"
-              placeholder="***********"
-              name="password"
-              onChange={event => this.handleChange(event, "password")}
-            />
+          <TextField
+          id="outlined-full-width"
+          label="Password"
+          style={{ margin: 8 }}
```
Comment (elit-altum) [inline src/auth/login-form/login-form.js]: I think your ESLint setup has caused some problems with the indentation. Please correct that. The closing tag and attributes aren't properly indented.

Comment (rak-shit) [reply src/auth/login-form/login-form.js]: Can you see it now.

File: src/auth/login/login.scss (hunk 1 above)
Comment (rak-shit) [reply src/auth/login/login.scss]: I have made the changes.

**Pull request closed without merging**